src/client.js

```diff
@@ -175,6 +175,54 @@ function keyFromRecoverySession(session, decryptionKey) {
  * @param {boolean} [opts.fallbackICEServerAllowed]
  * Optional. Whether to allow a fallback ICE server should be used for negotiating a
  * WebRTC connection if the homeserver doesn't provide any servers. Defaults to false.
+ *
+ * @param {object} opts.cryptoCallbacks Optional. Callbacks for crypto and cross-signing.
+ *
+ * @param {function} [opts.cryptoCallbacks.getPrivateKey]
+ * Optional (required for cross-signing). Function to call when a private key is needed.
+ * Args:
```

Comment:
Does JSDoc do the right thing here, or does it need extra linebreaks to know that there are line endings?  JSDoc has a [`@callback`](https://jsdoc.app/tags-callback.html) tag, which may be useful here, but it's extra work and I don't know if the version of JSDoc that we use supports it.

Reply:
Possibly not, although I'm inclined to say as long as the info is there this is probably fine - we can tweak the jsdoc to get it right at a later date.